docs/guides/admin/docs/installation/source-linux.md

```diff
@@ -71,6 +71,13 @@ Required for animate service (optional):
 
     synfig
 
+
+Java version support
```

Comment:
Isn't this already in here a few lines above this:

> Please make sure to install the following dependencies.
> Required:
>
>    java-1.8.0-openjdk-devel.x86_64 / openjdk-8-jdk

Reply:
I deliberately added it again to emphasise that JDK 8, and only 8, is required.  That being said, your solution makes things even more clear, so I'll remove this part.  If you're installing from source and can't follow the directions closely enough then that's not really something we can fix :)